src/sentry/static/sentry/app/views/settings/components/forms/field/fieldControl.jsx

```diff
@@ -45,13 +45,13 @@ class FieldControl extends React.Component {
     alignRight: PropTypes.bool,
     disabled: PropTypes.bool,
     disabledReason: PropTypes.node,
-    hideControlState: PropTypes.bool,
+    flexibleControlStateSize: PropTypes.bool,
```

Comment:
this name was confusing to me - it means that the size is set by the parent?

Reply:
mmm yeah I wanted the opposite of `fixed` size. It means that control state doesn't have a fixed width.